event_handler/sources.py

```diff
@@ -84,27 +84,29 @@ def get_source(headers):
     Gets the source from the User-Agent header
     """
     if "X-Gitlab-Event" in headers:
-        return "Gitlab"
+        return "hitlab"
 
     if "Ce-Type" in headers and "tekton" in headers["Ce-Type"]:
-        return "Tekton"
+        return "tekton"
 
     if "User-Agent" in headers:
-        if "/" in headers["User-Agent"]:
+        if "GitHub" in headers["User-Agent"]:
```

Comment:
I think we can get rid of the rest of this logic since we're only matching known cases at this point.  We can just return that it's not an authorized source.

Reply:
So if "GitHub" in headers.get("User-Agent"): return GitHub

else: return headers.get("User-Agent") -- this will return a null value if User-Agent isn't a header.